booking/src/main/java/org/dcsa/conformance/standards/booking/party/DynamicScenarioParameters.java

```diff
@@ -16,7 +16,9 @@ public record DynamicScenarioParameters(
     String carrierBookingRequestReference,
     String carrierBookingReference,
     BookingState bookingStatus,
-    BookingState amendedBookingStatus) {
+    BookingState amendedBookingStatus,
+    JsonNode booking
```

Comment:
Are we not checking the amended booking request anywhere?

The `GET /booking/{ref}?amendedContent=true` (or whatever the query parameter is called)

Reply:
I am not sure what to check there, because the shipper modify the payload. The modified payload must be checked against the original Booking. If that's the case, then there would be some difference for sure w.r.t original booking and the checks fail. I tried this. Can you let me know what needs to done in order to check the amended Booking  request ?

Reply:
> I am not sure what to check there, because the shipper modify the payload. The modified payload must be checked against the original Booking. If that's the case, then there would be some difference for sure w.r.t original booking and the checks fail. I tried this. Can you let me know what needs to done in order to check the amended Booking request ?

If the shipper requests an amendment to the booking and then uses `GET` to see their request again, the carrier should not have changed the amended variant between the request and the `GET` call. This implies tracking the current booking request and the amended booking requests separately and picking the right one for comparison based on the scenario step.